examples/usecases/multi-gpu-data-parallel-training.ipynb

```diff
@@ -0,0 +1,469 @@
+{
```

Comment:
<p>take a look on --&gt; take a look at.</p><p> </p><p>what'd happen if we have a super large dataset and only 2 gpus? how we would set <code>out_files_per_proc</code>  arg then?  can it be multiply of the num_gpus like 4, 6 etc?  the reason I ask this, if we have a large set, we might want to output several parquet files.. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/855/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>what'd happen if we have a super large dataset and only 2 gpus? how we would set&nbsp;<code>out_files_per_proc</code>&nbsp;arg then? can it be multiply of the num_gpus like 4, 6 etc? the reason I ask this, if we have a large set, we might want to output several parquet files..</p><p> </p><p>That is correct</p>